src/types/semi-partial-tezos-operation.mock.ts

```diff
@@ -0,0 +1,85 @@
+import { TezosOperationType } from '@airgap/beacon-sdk';
```

Comment:
maybe move this file in mocks folder?

Reply:
I think there is absolutely no need to do it